packages/neptune-extension/src/common/components/App.tsx

```diff
@@ -27,7 +32,7 @@ const App = ({
     const checkpoint = await leaderboardClient.api.createCheckpoint2({
       notebookId: notebook.id,
       checkpoint: {
-        path: metadata.path,
+        path: metadata.path || '',
```

Comment:
If there is no metadata.path or no metadata at all, we should not make any requests in the first place.